File: UI/WEB/src/Models/PaginationModel.ts

export default class PaginationModel {
    constructor() {
        this.PageSize = 3;
        this.CurrentPage = 1;
    }
    SortColumnName: string = "";
    SortOrder: string = "asc";
    PageSize: number = 0;
    CurrentPage: number = 0;
    TotalPages: number = 0;
    TotalRecords: number = 0;
}

File: API/KenJobs.Api/KenJobs.Api/Models/PaginationModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace KenJobs.Api.Models
{
    public class PaginationModel
    {
        public string SortColumnName { get; set; }
        public string SortOrder { get; set; }
        public int PageSize { get; set; }
        public int CurrentPage { get; set; }
        public int TotalPages { get; set; }
        public int TotalRecords { get; set; }
    }
}